CONTRIBUTING.md

```diff
@@ -175,7 +175,7 @@ in order to craft an excellent pull request:
    git push origin <topic-branch-name>
    ```
 
-7. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/)
+7. [Open a Pull Request](https://help.github.com/articles/about-git-rebase/)
```

Comment:
https://help.github.com/articles/about-pull-requests/ is not a 301 and is a different article than https://help.github.com/articles/about-git-rebase/ . What am I missing? :)